server/knexfile.ts

```diff
@@ -1,19 +1,18 @@
 import { Knex } from "knex";
-import dotenv from 'dotenv';
-dotenv.config();
+import {  dbHost, dbUser, dbPassword, dbName, dbDialect } from './src/constants.js';

```

Comment:
nice

Reply:
So I only did this because I'd already created constants.ts but never actually implemented it.
I think in a previous project I was having an issue with compiling TS and this was the only way I could get my ENV variables, so I just did it again here without thinking about it. I don't even use an envrionment.d.ts ([as seen here](https://stackoverflow.com/a/53981706)) but I guess this way is an alternative to that?

Is that why this is better? Is there another reason?

Reply:
You could also add it to environment.d.ts, but that's just describing the types not declaring the variables. My nice was around using constant files, which are great when your code gets bigger.